pkg/helm/helm.go

```diff
@@ -8,16 +8,22 @@ import (
 
 // Uninstall uninstalls a helm chart of the given name. There is no error in the case
 // of the helm chart not existing.
-func Uninstall(chartName string) error {
-	helmArgs := []string{"uninstall", chartName}
+func Uninstall(chartName string, namespace string) error {
+	helmArgs := []string{"uninstall", chartName, "-n", namespace}
 	return executeHelmCommand(helmArgs, isNotFoundMessage)
 }
 
 // Install a helm chert at the given path with the given name and the provided set arguments.
-func Install(chartPath, chartName string, args map[string]string) error {
-	helmArgs := []string{"install", "--skip-crds"}
-	helmArgs = append(helmArgs, mapToHelmArgs(args)...)
+func Install(chartPath, chartName string, flags map[string]string, values map[string]string) error {
```

Comment:
do we need to pass two different sets of flags? I think this could be a single map right? I believe the current `values` map could be refactored slightly to pass `--set` with the value to the new `flags` map with the new approach. What do you think?

Reply:
as discussed on zoom, I will rename values to templateValues and keep 2 maps for keeping a clear distinction between helm arguments and template arguments